packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -61,8 +62,9 @@ const seenTxKeyOf = evidence => {
  * XXX consider separate facets for `Advancing` and `Settling` capabilities.
  *
  * @param {Zone} zone
+ * @param {() => Promise<StorageNode>} makeStatusNode
  */
-export const prepareStatusManager = zone => {
+export const prepareStatusManager = (zone, makeStatusNode) => {
```

Comment:
We also need to call `recordStatus` in `advanceOutcome` - can use `txpost`

Reply:
Done